.github/workflows/deployment.yaml

```diff
@@ -16,6 +16,11 @@ jobs:
       - uses: webfactory/ssh-agent@v0.5.3
         with:
           ssh-private-key: ${{ secrets.GH_PAGES_DEPLOY }}
+      - name: Generate Seneca-IPC144.pdf
+        run: |
+          chmod +x ./convert-to-pdf.sh
```

Comment:
In a Unix shell, please do:

```sh
chmod +x ./convert-to-pdf.sh
```

Then you can git add and commit the file, add the executable mode will be retained.

Reply:
This is still not done.  Please do what I asked above, you don't need to do this once it's set in git.

Reply:
You keep saying you're done, but every time I come back, this is not done.  Please do this before you ask for another review.

Reply:
> You keep saying you're done, but every time I come back, this is not done. Please do this before you ask for another review.
I
Sorry, I added 'add  and commit' but its in the next line so maybe that's why github didnt mark this one outdated?